acceptance/lib/puppet/acceptance/common_utils.rb

```diff
@@ -6,5 +6,111 @@ def ruby_command(host)
       end
       module_function :ruby_command
     end
-  end
-end
+
+    module CAUtils
+      def initialize_ssl
+        hostname = on(master, 'facter hostname').stdout.strip
+        fqdn = on(master, 'facter fqdn').stdout.strip
+
+        if master.use_service_scripts?
```

Comment:
You may need to specify to use service scripts like we do in puppet's presuite: https://github.com/puppetlabs/puppet/blob/master/acceptance/setup/aio/pre-suite/015_PackageHostsPresets.rb